### ISSUE TITLE
Release churn

### ISSUE THREAD

**Issue by stack72** [okta/terraform-provider-okta#487]
Hi

I have been receiving multiple RSS feed updates from this repository regarding that fact that v3.11.0 of the package was released. I received updates as follows:

* 26th March
* 13th May
* 20th May
* 21st May
* 25th May

This leads me to believe that the published packages are not stable for use - I can see only 1 GitHub 3.11.0 release - did you remove the other packages? If so, for what reason? Writing the same release over and over again is not useful for an end user as it means we don't have faith on what was released

I can also see that v3.11.1 was "released" yesterday as per RSS feed and is now removed

Paul

**Comment by bogdanprodan-okta** [comment]
Hi, @stack72 ! Yes, these we basically unsuccessful attempts to publish the provider into a new namespace. Currently, version 3.11.1 is the latest one and should be working
```
required_providers {
    okta = {
      source = "okta/okta"
      version = "3.11.1"
    }
  }
```
I've asked Hashicorp to add an old `gpg` key as well, so all the previous versions should be valid as well.

**Comment by stack72** [comment]
But 3.11.1 was already published yesterday... I think you may need to abandon any 3.11.x releases and just create a new 3.12.0 release as this has not had any other published artifacts for it. Re-releasing the same artifact version is something that should be avoided - not everyone uses this tool from the Terraform registry

I am looking forward to this being resolved so I can take advantage of the v3.12.0 release when that happens
